src/lib/holocene/table/paginated-table.svelte

```diff
@@ -180,7 +180,9 @@
     @apply surface-primary;
 
     :global(tr:not(.empty)) {
-      @apply h-12 border-b border-primary last-of-type:border-0 hover:bg-gradient-to-br hover:from-blue-100 hover:to-purple-100 hover:bg-fixed;
+      @apply h-12 border-b border-table last-of-type:border-0;
+      @apply dark:hover:bg-gradient-to-br dark:hover:from-indigo-100 dark:hover:to-indigo-700 dark:hover:bg-fixed dark:hover:text-inverse;
+      @apply hover:surface-subtle dark:hover:text-inverse;
```

Comment:
Can these be combined? 

```suggestion
      @apply h-12 border-b border-table hover:surface-subtle last-of-type:border-0 dark:hover:bg-gradient-to-br dark:hover:from-indigo-100 dark:hover:to-indigo-700 dark:hover:bg-fixed dark:hover:text-inverse dark:hover:text-inverse;
```

Reply:
Sure! I changed this to be more human-readable but I will change it back!